packages/steveo/test/producer/sqs_test.ts

```diff
@@ -165,6 +176,69 @@ describe('SQS Producer', () => {
       ).to.equal(attributes[0].value);
     });
 
+    it('should merge the context object into payload metadata if context given', async () => {
```

Comment:
Could we also have the same test for kafka?

Reply:
@tjcoyoca Kafka tests added [above](https://github.com/ordermentum/steveo/pull/788/files#diff-252d51846e4837617e7e69b7d9c6bbbcf912b960234a1908b7f1f2d14ebea915R59)